includes/admin/class-wcmp-export.php

```diff
@@ -905,12 +905,11 @@ public function getOrderShippingClass(WC_Order $order, string $shippingMethodId
      */
     public function getPackageTypeFromOrder(WC_Order $order, AbstractDeliveryOptionsAdapter $deliveryOptions = null): string
     {
-        $packageTypeFromDeliveryOptions = $deliveryOptions
-            ? $deliveryOptions->getPackageType()
-            : null;
+        $packageTypeFromDeliveryOptions = $deliveryOptions ? $deliveryOptions->getPackageType() : null;
+        $correctPackageType             = $this->getAllowedPackageType($order, $packageTypeFromDeliveryOptions);
```

Comment:
Waarom correct? Wat is er correct aan?

Reply:
correct / allowed eigenlijk. Het kijkt of de juiste packagetype wordt geselecteerd voor de juiste bestemming.

Reply:
aangepast